httpx/_content_streams.py

```diff
@@ -298,8 +281,8 @@ def iter_fields(
             else:
                 yield self.DataField(name=name, value=value)
 
-        for name, value in files.items():
-            yield self.FileField(name=name, value=value)
+        for name, filevalue in files.items():
+            yield self.FileField(name=name, value=filevalue)
```

Comment:
Different variable name to `value` is required of mypy would complain that the `value` in `data` doesn't have the same type as the `value` in `files` (but ideally we don't want to just use `Any` here).